### PR TITLE
[v1.5] Add ACI CNI 6.0.4.1 variables

### PR DESCRIPTION
**Pull request by jeffinkottaram**
https://github.com/rancher/rancher/issues/44627

### CODE REVIEW

**Review by snasovich — APPROVED** [review]

**Review by chiukapoor — APPROVED** [review]

**Comment by jiaqiluo** [comment]
This is just a reminder that, either now or after 1.5.6 is released, those changes should be backported to the `release/v1.5` branch.

**Comment by kinarashah** [comment]
@akhilesh-oc Could you re-run `go generate`? KDM PR is merged so we can update data.json with this PR before merging it.

**Comment by akhilesh-oc** [comment]
> @akhilesh-oc Could you re-run `go generate`? KDM PR is merged so we can update data.json with this PR before merging it.

Should we wait till KDM PR gets merged to dev-v2.8 for generating data ?
which URL should we update to generate data?

**Review by snasovich — APPROVED** [review]

**Review by chiukapoor — APPROVED** [review]

**Pull request merged**
